sklearn_numba_dpex/kmeans/tests/test_kmeans.py

```diff
@@ -364,7 +366,7 @@ def _get_score_with_centers(centers):
         [
             -1827.22702,
             -1027.674243,
-            -865.257397,
+            -865.257501,
```

Comment:
The difference in those values comes from numerical instability after including the X_mean removal step.